node/basic-cloud-functions-bot/index.js

```diff
@@ -18,190 +18,196 @@
+    'https://photos.app.goo.gl/DRqnKP4YBO12SnLT2',
   ];
   return {
     cards: [{
         header: {
-          title: "Which type",
-          subtitle: "Click on to Choose",
+          title: 'Which type',
+          subtitle: 'Click on to Choose',
           imageUrl:
-            "http://www.freepngimg.com/download/dog/9-dog-png-image-picture-download-dogs.png",
-          imageStyle: "IMAGE"
+            'http://www.freepngimg.com/download/dog/9-dog-png-image-picture-download-dogs.png',
+          imageStyle: 'IMAGE',
         },
         sections: [{
-            widgets: getWidgets(imageUrls, getImageButtonWidget)
-          }]
-      }]
+            widgets: getWidgets(imageUrls, getImageButtonWidget),
+          }],
+      }],
   };
 }
 
 /**
- * Return an array of widgets
+ * Return an array of widgets.
  * @param {Array} urls a simple array of imageUrls, strings
  * @param {Function} widgetComposer a function used to compose a widget object
- * @returns {Array}
+ * @return {Array}
  */
 function getWidgets(urls, widgetComposer) {
-  return urls.map(url => widgetComposer(url));
+  return urls.map((url) => widgetComposer(url));
 }
 
 /**
- * Returns a Widget with an image and button with a set url parameter
+ * Returns a Widget with an image and button with a set url parameter.
  * @param {String} imageUrl
- * @param {String} actionMethodName a simple string that can be passed to choose what actionMethodName is to be specified
+ * @param {String} actionMethodName a simple string that can be passed to choose
+ * what actionMethodName is to be specified
  * @return {Object} a widget with an image and button
  */
-function getImageButtonWidget(imageUrl, actionMethodName = "choose") {
+function getImageButtonWidget(imageUrl, actionMethodName = 'choose') {
   return {
     image: {
-      imageUrl
+      imageUrl,
     },
     buttons: [{
         textButton: {
-          text: "Select",
+          text: 'Select',
           onClick: {
             action: {
               actionMethodName,
               parameters: [{
-                  key: "url",
-                  value: imageUrl
-                }]
-            }
-          }
-        }
-      }]
+                  key: 'url',
+                  value: imageUrl,
+                }],
+            },
+          },
+        },
+      }],
   };
 }
 
 /**
- * Returns a Single Card with a Single Image
+ * Returns a Single Card with a Single Image.
  * @param {String} imageUrl
- * @returns {Object}
+ * @return {Object}
  */
 function getSingleImage(imageUrl) {
   return {
     cards: [{
         sections: [{
             widgets: [{
                 image: {
-                  imageUrl
-                }
-              }]
-          }]
-      }]
+                  imageUrl,
+                },
+              }],
+          }],
+      }],
   };
 }
 
 /**
- * Returns a Card List Commands/Usage of the bot
- * @returns {Object}
+ * Returns a Card List Commands/Usage of the bot.
```

Comment:
`@param`

Reply:
Not looking to improve javadoc in this PR.